### DISCUSSION TITLE
Mnemonic is invalid

### DISCUSSION THREAD

**Discussion by Yulya94** [iotaledger/firefly#2538] [Help & Support]
Hello. Can't login to Firefly Wallet. There are 24 words to restore, but when they are entered, it says that the Mnemonic is invalid. What does it mean?

**Comment by MiKeF05** [comment]
It means that your 24 words are not correct. Have you noted them correctly? Is the order correct?

**Reply by Yulya94** [reply]
Yes. I numbered all the words when I wrote them down. The words are spelled correctly. What else can I do to recover my wallet?
